infrastructure/log/log.go

```diff
@@ -0,0 +1,57 @@
+package log
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/utahta/momoclo-channel/domain/event"
+	"google.golang.org/appengine/log"
+)
+
+type logger struct {
+	ctx context.Context
+}
+
+func NewLogger(ctx context.Context) event.Logger {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function NewLogger should have comment or be unexported